src/config.json

```diff
@@ -958,9 +958,9 @@
       }
     },
     "server": {
-      "url": "http://localhost:9999/arlas",
+      "url": "http://demo.arlas.io/arlas",
```

Comment:
should we keep `http://localhost:9999/arlas` and `geodata` ?

Reply:
Fixed